dex/src/test/scala/com/wavesplatform/dex/db/WithDb.scala

```diff
@@ -11,11 +11,12 @@ import org.iq80.leveldb.{DB, Options}
 import org.scalatest.{BeforeAndAfterEach, Suite}
 
 import java.nio.file.Files
-import scala.concurrent.Future
+import java.util.concurrent.Executors
+import scala.concurrent.{ExecutionContext, Future}
 
-trait WithDB extends BeforeAndAfterEach { this: Suite =>
+trait WithDb extends BeforeAndAfterEach { this: Suite =>
 
-  implicit protected val executionContext = scala.concurrent.ExecutionContext.Implicits.global
+  implicit private val ec: ExecutionContext = ExecutionContext.fromExecutor(Executors.newSingleThreadExecutor())
```

Comment:
Please make it daemonic, e.g.:
```scala
  Executors.newSingleThreadExecutor(new ThreadFactoryBuilder().setDaemon(true).build())
```
otherwise these thread pools stay in the SBT session (we don't fork tests)